### PR TITLE
Bump grpc version to support build on apple m1

### PR DESCRIPTION
**Pull request by Shoothzj**
tested on my m1 mbp

For additional information, we can see
https://github.com/apache/pulsar/pull/13076
https://github.com/apache/pulsar/pull/16306

### CODE REVIEW

**Comment by imbajin** [comment]
Thanks for your contribution, is there anyway to test the diff (or ensure it's fine on m1)?

@seagle-yuan any suggestion could we test it?

**Comment by Shoothzj** [comment]
@imbajin Sorry, I forget to fill how I test it. I have test it on my mbp. It can package well. 
For additional information, we can see
https://github.com/apache/pulsar/pull/13076
https://github.com/apache/pulsar/pull/16306

File: hugegraph-core/src/protobuf/java/com/baidu/hugegraph/backend/store/raft/rpc/RaftRequests.java [hunk 1]
```diff
@@ -60,13 +60,19 @@ public final int getNumber() {
     }
 
     /**
+     * @param value The numeric wire value of the corresponding enum entry.
```
Comment (javeme) [inline hugegraph-core/src/protobuf/java/com/baidu/hugegraph/backend/store/raft/rpc/RaftRequests.java:63]: can we not update RaftRequests.java file?

**Review by javeme — COMMENTED** [review]
Thanks, one tiny comment

**Comment by Shoothzj** [comment]
@javeme I think we should update the `RaftRequests.java`. It's generated by grpc. We should keep it consistent.
We'd better not commit this file to the repository, but that's another optimization point.

**Review by javeme — APPROVED** [review]

**Comment by Shoothzj** [comment]
can this pr be merged? what should I do next? @javeme @imbajin

**Review by imbajin — APPROVED** [review]

**Pull request merged**